02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>Add in a sentence to say why it's useful to have multi-level hierarchies and not lots of single level ones</p><p>You could mention the UI and the you'll be able to drill into the data or aggregate to only a given level</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>